module/move/wca/src/ca/help.rs

```diff
@@ -12,6 +12,7 @@ pub( crate ) mod private
   use error_tools::for_app::anyhow;
   use former::Former;
   use ca::tool::table::format_table;
+  use crate::ca::aggregator::private::Order;
```

Comment:
This seems to be not in line with the [convention](https://github.com/obox-systems/conventions/blob/master/code_style.md#importing-local-entities)